backend/db/migrate/20240807220910_delete_index_from_user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
Remember me, please, why you have decide to delete migration, that have add this index add create new migration, removed this index?